### ISSUE TITLE
Consider removing/revising BAC for pedestrians on repeat fatalities. 

### ISSUE THREAD

**Issue by jaronheard** [hackoregon/transportation-frontend2#9]
I think you should consider removing the BAC of the pedestrians that were killed when cars hit them, or at least calling out in the footnote that it is not illegal to be drunk as a pedestrian. In my reading, the wording has the implication that the pedestrians were at fault in the collision, and I think that is unfair to the people that died.

I do think it's fair to call out the drunk drivers that were breaking the law and irresponsibly killed another human as the result of their actions.

I'm happy to make a pull request with my suggested changes.

**Comment by jaronheard** [comment]
The above is strongly worded because I care strongly about this issue, hope it did not come off as too critical. You all did an awesome job on this project!

**Comment by sanjuroj** [comment]
Nope, not too critical, feedback is always appreciated.  I think Lee and/or April will weigh in on this when the get a chance.  They worked on that part of the project much more than I did.

**Comment by meganmckissack** [comment]
Thanks for the feedback, Jaron.  I'm going to forward this to April and Lee because I don't think they're on github.

**Comment by znmeb** [comment]
It would only take me about a minute to invite them to GitHub ;-)
